### ISSUE TITLE
When clicking max ETH, subtract 0.01 gas fees.

### ISSUE THREAD

**Issue by piersss** [airswap/airswap-web#504]

**Comment by dmosites** [comment]
Makes sense! Figure it'd be hard for us to calculate the exact amount in the app since gas settings can be adjusted in the wallet. If 0.01 generally seems like the right amount that sounds good to me.

**Comment by piersss** [comment]
Talked to Jop about this. He will think about how to show this to the user visually.

**Comment by piersss** [comment]
My design proposal:
![Screenshot 2022-06-07 at 22 14 41](https://user-images.githubusercontent.com/86911296/172474559-a316e9e3-43f5-4d3a-a71d-138a990e10fc.png)

**Comment by piersss** [comment]
Or maybe remove tooltip and always show the balance of active token in top right.

**Comment by piersss** [comment]
Adding the ETH balance in top right will make it clear to the user that not all ETH is used after clicking MAX.

![image](https://user-images.githubusercontent.com/86911296/172707605-8965a2eb-c213-47e6-8e8a-de2020d12b05.png)
